package.json

```diff
@@ -5,8 +5,9 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "build-prod": "ng build -c production",
+    "build-prod": "\"$npm_execpath\" run collect-licenses && ng build -c production",
```

Comment:
Would this work for windows environments?

Reply:
On powershell, probably, but TIL that if you're using batch then it wants the `%npm_execpath%` syntax... Sigh

I'll remove this and just call `npm run collect-licenses` manually in the docker image instead.